docker-compose.yml

```
@@ -3,18 +3,40 @@
 version: '3'
 
 services:
+
+  main-db:
+    image: postgres:14-alpine
+      # Docs for options to the postgres server command:
+    # https://www.postgresql.org/docs/current/app-postgres.html
+    command: postgres -c "log_lock_waits=on" -N 1000 -c "fsync=off"
+    env_file: ./app/local.env
+    ports:
+      - "5432:5432"
+    platform: linux/amd64 # Fix running on Apple silicon
+    volumes:
+      - dbdata:/var/lib/postgresql/data
+
   # Define one service for the application.
-  app:
+  main-app:
     # This service relies on building from a Dockerfile in this file directory,
     # not on an image repository.
     build: ./app
     # Expose the application port for local development.
     ports:
       - 8080:8080
+    env_file: ./app/local.env
+    environment:
+      # NOTE: This takes precendence over the value specified
+      # in local.env as we can't use localhost to get to the docker
+      # container and instead use the alias defined above
+      - DB_HOST=main-db
+    platform: linux/amd64 # Fix running on Apple silicon
     # Mount application directory for local development.
     volumes:
       - ./app:/srv
+    depends_on:
+      - "main-db"
```

Let's go no-quotes to be consistent with the other lines.
```suggestion
      - main-db
```